README.md

```diff
@@ -1,142 +1,88 @@
-balena CLI
-=========
+# balena CLI
```

Comment:
I have added some code to `capitanodoc.ts` to automatically extract selected sections from README.md to generate the markdown for the website. The parser looks at the hash (`#`) characters at the beginning of a line to identify markdown sections, and for this reason I replaced the notation here.